apps/01PigLatin.js

```diff
@@ -64,3 +120,4 @@ if (typeof describe !== 'undefined') {
     getPrompt();
 
 }
```

Comment:
Thanks- I'll read more on MDN and update my answers.

Sent from my iPhone

> On Aug 14, 2016, at 4:11 PM, maddiesolz <notifications@github.com> wrote:
> 
> In apps/01PigLatin.js:
> 
> > @@ -64,3 +120,4 @@ if (typeof describe !== 'undefined') {
> >      getPrompt();
> >  
> >  }
> Good job on the app, but I am not sure based off your answers that you completely understand how the app is working. Read some of the links to the MDN documentation that is provided in the comments of this code and try to fix you answers at a later time. I will give you full credit for this assignment for now.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>